pep-9999.rst

```diff
@@ -256,7 +256,11 @@ with two underscores), it can be used as a keyword argument [#pep484]_::
 PEP 570 [#pep570]_ style positional-only parameters are currently not
 supported.
 
-If an argument is unannotated, its type is assumed to be ``Any``. The type of
+If an argument or return type is unannotated, per PEP 484 [#pep484]_ its
+type is assumed to be ``Any``. Some type checkers can optionally treat
+unannotated arguments or return types more strictly.
+
```

Comment:
How about mentioning the recommendation to not annotate unknown types here? Something like: "Note that it is preferred to leave unknown types unannotated rather than explicitly marking them as `Any`, as some type checkers can optionally ..." I know this already covered later on, but I don't think there's any harm in repeating it, and it'll reinforce that Any and unannotated are not interchangeable.

Reply:
+1

I know that mypy has some options that warn about arguments without annotations. We should make it clear that the way to shut those up is to either not use those options on stubs or to change mypy to not complain about this in stubs, *not* to add `Any` to the stubs.